src/locales/en.json

```diff
@@ -13,7 +13,9 @@
 
   "menu": {
     "apps": "Apps",
-    "settings": "Settings"
+    "settings": "Settings",
+    "home_mobile": "Back to home...",
```

Comment:
Are `...` necessary ? It feels weird reading it here. Maybe in the UI it feels better.

Reply:
I follow the mockup for this text, I think we will discuss again about that wording